APMrover2/APMrover2.cpp

```diff
@@ -75,9 +75,19 @@ const AP_Scheduler::Task Rover::scheduler_tasks[] = {
     SCHED_TASK(accel_cal_update,       10,    100),
     SCHED_TASK(dataflash_periodic,     50,    300),
     SCHED_TASK(button_update,          5,     100),
+    SCHED_TASK(stats_update,           1,     100),
 };
 
 /*
+  update AP_Stats
+*/
+void Rover::stats_update(void)
+{
+    g2.stats.set_flying(motor_active());
```

Comment:
Yes, this one caused a bit of pain, but I couldn't find something closer that I was happy with.
run_timer is ambiguous; it's pretty close to the other timer, the "total uptime" of the autopilot
is_moving is wrong; it might be moving for some reason other than its motors are running (e.g. on a boat)

set_vehicle_active is closer, but "active" in the mavlink context pretty much means "armed", and I didn't want confusion there (the intent of this flag is the vehicle is actively undertaking movement itself).  Of course, you can be flying without meaning to, but it is unlikely.

is_actively_moving / is_intentionally_moving?  I *do* need to fit this into a parameter name somehow :-)